test/verify/check-apps

```diff
@@ -174,11 +174,11 @@ class TestApps(packagelib.PackageCase):
         b.wait_not_present("#refresh-progress")
         b.wait_visible(".pf-v5-c-empty-state")
 
-        # known OS: appstream-data-tst gets installed from the map
+        # known OS: appstream-data-test gets installed from the map
         m.write("/etc/os-release", 'ID="derivative"\nID_LIKE="spicy testy"\nVERSION_ID="1"\n')
         b.click("#refresh")
-        with b.wait_timeout(30):
-            b.wait_visible(".app-list #app-1")
+        m.execute("until test -e /usr/share/app-info/xmls/test.xml; do sleep 1; done")
```

Comment:
What about using `b.wait` for the loop? Then we can control the timeout in the usual way and don't need to wait for the global timeout when this fails.

Reply:
m.execute() has a default timeout of 2 minutes; it can be changed with `timeout=`, but the default seemed quite appropriate to me. We often use this pattern in our tests, see `git grep -E 'execute.*(while|until)'`

Reply:
and `wait()` (not `b.wait()` !) is more expensive, as it creates new SSH connections for each iteration.

Reply:
> m.execute() has a default timeout of 2 minutes

Ah, I didn't think of that!